packages/ripple-address-codec/package.json

```diff
@@ -21,12 +21,13 @@
   "prepublishOnly": "tslint -b ./ && jest",
   "scripts": {
     "build": "tsc -b",
-    "test": "jest",
+    "test": "jest --verbose false --silent=false --runInBand --detectOpenHandles ./src/*.test.js",
     "lint": "eslint . --ext .ts",
     "clean": "rm -rf ./dist && rm -rf tsconfig.tsbuildinfo"
   },
   "prettier": "@xrplf/prettier-config",
   "engines": {
-    "node": ">= 10"
+    "node": ">=14",
+    "npm": ">=7.0.0 < 8.0.0"
```

Comment:
The npm requirements shouldn't be listed - https://github.com/XRPLF/xrpl.js/pull/1848

Reply:
Nice catch